ClusterDestroyer/ClusterDestroyer.py

```diff
@@ -28,17 +28,24 @@ def destroyServers(clusterDictionary):
                                project=str(os.environ["PROJECT"]),
                                datacenter=str(os.environ["ZONE"]))
         nodeList = []
-
+        delNames = []
+        # get a list of nodes from the Cloud Provider
         nodes = driver.list_nodes(ex_zone=str(os.environ["ZONE"]))
+        numNodes = int(clusterDictionary["nodeQty"])
+        # Create a list of node names to delete
+        for i in range(0, int(numNodes)+1):
```

Comment:
If my previous comment is true, I think the `+1` in the `int(numNodes)+1` is not needed.  If we have 5 nodes, it's correct that we'll have 000 to 004 for `range(0, 5)` with the current google cloud provisioning implementation.